### ISSUE TITLE
Why Baseline (OC) Research & Technology Transfer & Extension & Proximity Services District Level Annual Dataset is set as yearly?

### ISSUE THREAD

**Issue by Aleglise** [minagri-rwanda/DHIS2-Agriculture#110]
Baseline data sets should be set with Baseline attribute = "yes"?

@abyot please explain again @arunimam  how data sets should be design.

If there are so;e change in the configuration, please inform us.

Thanks

**Comment by abyot** [comment]
I am not sure what I am supposed to explain here. I also do not know which of the data sets should be made yearly or quarterly.

But one thing we agreed and also designed the system is that for those data sets where we have quarterly target and progress data collected. We make their baseline data set quarterly also. But since data is collected only once in a year, we designed the data entry screen in such away that their period is displayed (**only in data entry UI**) as yearly but internally we store the value as quarterly.

To treat such baseline data sets differently - as mentioned above - we have created a special property attribute called "Baseline" with value either "Yes" or "No". Having this property set to "Yes" is crucial for the above effect to take place.
